protoc_plugin/lib/src/generated/dart_options.pb.dart

```diff
@@ -171,7 +172,7 @@ class Imports extends $pb.GeneratedMessage {
   ///  so the generated code may contain errors. Therefore, running dartanalyzer
   ///  on the generated file is a good idea.
   @$pb.TagNumber(1)
-  $core.List<DartMixin> get mixins => $_getList(0);
+  $pb.PbList<DartMixin> get mixins => $_getList(0);
```

Comment:
I'm not sure why this change. I assume the files have not been regenerated in the past.